web/webkit/src/main/scala/net/liftweb/http/provider/servlet/HTTPRequestServlet.scala

```diff
@@ -234,8 +236,8 @@ private class OfflineRequestSnapshot(req: HTTPRequest, val provider: HTTPProvide
 
   val scheme: String = req.scheme
 
-  lazy val serverPort: Int = req.serverPort match {
-    case 80 => headers("X-SSL").flatMap(Helpers.asBoolean _).filter(a => a).map(a => 443).headOption getOrElse 80
+  lazy val serverPort: Int = _serverPort match {
+    case 80 => headers("X-SSL").flatMap(Helpers.asBoolean).filter(identity).map(a => 443).headOption getOrElse 80
```

Comment:
Formatting done. 

And about that `filter` call: It seems to exist because the `headers` method returns a `List[String]` and we want to check if there is any `true` value.

But, now that I looked into it a bit more, I think there might be something wrong with how the `headers` method is used here. It doesn't return header values, it just returns the names of the headers, which are never going to be converted to a `true` value and the result of this entire chain is always 80.

I am not sure if I should push a change here or create a ticket.